recipe/meta.yaml

```diff
@@ -12,28 +12,28 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win]
 
 requirements:
   build:
     - rust
     - pkg-config
     - fontconfig 2.12.*  # [not osx]
-    - freetype 2.7
+    - freetype 2.8.*
     - graphite2 1.3.*
-    - harfbuzz 1.4.*  # note: conda-forge pins 1.3, but we need 1.4
```

Comment:
@pkgw is this OK? If not you can close this PR b/c we don't have `harfbuzz` compiled with the other dep version pinned here. (I can try to do that later though.)

Reply:
Upgrading to 1.7 should definitely be OK, I think. The CI failure looks like the SSL problem that has been cropping up.